helpdesk_mgmt_fieldservice_equipment/tests/test_fsm_equipment.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestFSMEquipment(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Create test data
```

Comment:
```suggestion
        cls.env = cls.env(context=dict(cls.env.context, **DISABLED_MAIL_CONTEXT))
        # Create test data
```
> Disable tracking by setting tracking_disable on the whole test env (eg: `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))` ) or evaluate the use of `BaseCommon` as base test class if you need to setup company, currency, users and groups in a special way)

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0